### DISCUSSION TITLE
How to build a js-library with react-native-reanimated/plugin

### DISCUSSION THREAD

**Discussion by little-buddy** [callstack/react-native-builder-bob#456] [Q&A] [answered]
I have added plugin to babel.config.js

<img width="490" alt="image" src="https://github.com/callstack/react-native-builder-bob/assets/30054881/887d3e6b-5c37-4ca3-b20e-650cd851b4c2">

But LogBox told me it wasn't transformed with `worklet`

<img width="378" alt="image" src="https://github.com/callstack/react-native-builder-bob/assets/30054881/741230af-b448-4f8d-8f1a-07ff61e14769">

**Comment by satya164** [comment, marked as answer]
The app you're running is the example app, so you need to add the babel plugin there.

**Comment by little-buddy** [comment]
Thanks for your reply. 
I found that I did not add `GestureHanlderRootView` in the example, 
but it always reported react-native-reanimated error, 
which confused me
